requirements.txt

```diff
@@ -35,16 +35,16 @@ python-gnupg==0.3.9
 python-json-logger
 python-keyczar==0.716
 pytz
-pyyaml<4.0,>=3.11
+pyyaml<4.0,>=3.12
 rednose
-requests[security]<2.12,>=2.11.1
+requests[security]<2.14,>=2.13.0
```

Comment:
The latest version is 2.14 and even 2.14.1, see changelog: https://github.com/kennethreitz/requests/blob/master/HISTORY.rst

Can we pin to latest instead? That will fix st2cicd.